.eslintrc.js

```diff
@@ -33,6 +33,8 @@ module.exports = {
     'jsx-a11y/alt-text': ['warn'],
     'jsx-a11y/aria-role': ['warn'],
     'jsx-a11y/media-has-caption': ['off'],
+    'cypress/no-unnecessary-waiting': ['off'],
+    'cypress/unsafe-to-chain-command': ['warn'],
```

Comment:
I do not think we should disable/downgrade the error level of these on a project level. If there is a specific place this cannot be fixed, we should add a comment for es-lint to ignore it there, but our priority should be to fix these.
Adding waiting time to test can introduce unexpected behaviour in tests.

Reply:
I spoke with Chris about this and he wanted this added in.

Reply:
@chrisfpcc We shouldn't disable linting in order to preserve test reliability. @Cara-Barter Please remove, thanks!